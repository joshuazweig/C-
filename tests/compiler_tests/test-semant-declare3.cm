int main() {
    int i;
    for (i = 0; i < 5; i=i+1) {
        int x;
        x = 3 + i*i;
        printf("%d\n", x);
    }
    int x; 
    x = 6;
    printf("%d\n", x);
}
