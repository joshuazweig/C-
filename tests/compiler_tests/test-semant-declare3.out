3
4
7
12
19
6
